clinics/merge/patients.go

```diff
@@ -254,11 +258,14 @@ func (p *PatientMergePlanner) Plan(ctx context.Context) (PatientPlans, error) {
 }
 
 func (p *PatientMergePlanner) getTargetPatientById(userId string) (*patients.Patient, error) {
-	patient, ok := p.targetByUserId[userId]
-	if !ok || patient == nil {
+	pts := p.targetByAttribute[PatientAttributeUserId][userId]
+	if len(pts) == 0 || pts[0] == nil {
```

Comment:
Is there ever a time when you'd get a nil patient? Or is that just a remnant from the previous code that should probably be removed?

If you did have a nil, and it was intentional, what is that supposed to mean?